tools/model_tools/src/openvino/model_zoo/_configuration.py

```diff
@@ -226,67 +233,70 @@ def load_models(models_root, args):
-                composite_model_names.add(composite_model_name)
+                    if len(model_stages) == 0:
+                        continue
+                    subdirectory = composite_model_config.parent.relative_to(models_root)
+                    composite_models.append(CompositeModel.deserialize(
+                        composite_model, composite_model_name, subdirectory, model_stages
+                    ))
 
-    for config_path in sorted(models_root.glob('**/model.yml')):
-        subdirectory = config_path.parent
+                    if composite_model_name in composite_model_names:
+                        raise validation.DeserializationError(
+                            'Duplicate composite model name "{}"'.format(composite_model_name))
+                    composite_model_names.add(composite_model_name)
 
-        is_composite = (subdirectory.parent / 'composite-model.yml').exists()
-        if is_composite:
-            continue
+    if mode != ModelLoadingMode.composite_only:
+        for config_path in sorted(models_root.glob('**/model.yml')):
+            subdirectory = config_path.parent
 
-        subdirectory = subdirectory.relative_to(models_root)
+            is_composite = (subdirectory.parent / 'composite-model.yml').exists()
+            if is_composite and mode != ModelLoadingMode.ignore_composite:
+                continue
 
-        with config_path.open('rb') as config_file, \
-                validation.deserialization_context('In config "{}"'.format(config_path)):
+            subdirectory = subdirectory.relative_to(models_root)
 
-            model = yaml.safe_load(config_file)
-            if not schema.check(model):
-                raise validation.DeserializationError('Configuration file check was\'t successful.')
+            with config_path.open('rb') as config_file, \
+                    validation.deserialization_context('In config "{}"'.format(config_path)):
 
-            for bad_key in ['name', 'subdirectory']:
-                if bad_key in model:
-                    raise validation.DeserializationError('Unsupported key "{}"'.format(bad_key))
+                model = yaml.safe_load(config_file)
+                if not schema.check(model):
+                    raise validation.DeserializationError('Configuration file check was\'t successful.')
 
-            models.append(Model.deserialize(model, subdirectory.name, subdirectory, None))
+                for bad_key in ['name', 'subdirectory']:
+                    if bad_key in model:
+                        raise validation.DeserializationError('Unsupported key "{}"'.format(bad_key))
 
-            if models[-1].name in model_names:
-                raise validation.DeserializationError(
-                    'Duplicate model name "{}"'.format(models[-1].name))
-            model_names.add(models[-1].name)
+                models.append(Model.deserialize(model, subdirectory.name, subdirectory, None))
```

Comment:
We need to pass `composite_model_name`, in `ignore_composite` mode

Reply:
provided